integration_tests/timbre5/src/example/core.clj

```diff
@@ -15,4 +15,8 @@
     (.warn logger "Hello from SLF4J")
     (.info logger "Hello from SLF4J")
     (.debug logger "Hello from SLF4J")
-    (.trace logger "Hello from SLF4J")))
\ No newline at end of file
+    (.trace logger "Hello from SLF4J")))
+
+
+#_;
+        (-main)
```

Comment:
Is the call to `-main` necessary for the test?

Reply:
No, it just simplifies testing from a REPL. That's why I made the reader skip (-main) using the "#_" reader macro. The comment is just to ease readability in the editor.

Nothing essential, take it, leave it, it does not matter for functionality.

Reply:
If you'd rather keep it, you could put it in a rich comment, makes the intent clearer.